boards/xtensa/esp32/Makefile.board

```diff
@@ -1,4 +1,3 @@
-# Default to xt-run but let the user override to xtsc-run if they desire
-# TODO
 EMU_PLATFORM ?=
 DEBUG_SCRIPT :=
+FLASH_SCRIPT := esp32.sh
```

Comment:
the support scripts we have are designed to do more than flashing, so I think this is in line with other scripts we have, if we add debugging, then it is added to the same script.